### DISCUSSION TITLE
Human Genome Research & Important information

### DISCUSSION THREAD

**Discussion by LimesKey** [VerisimilitudeX/DNAnalyzer#291] [Ideas]
I took a look at [23AndMe's patent ](https://patents.google.com/patent/WO2004092333A2/en?oq=23andme)to see how they did the more complex stuff and I noted down some crucial parts from it. Most of the information is biological information. I am still not certain on how we can use this data because of its patent.

### Information found

> "Even SNPs that occur in non-coding regions can result in altered mRNA and/or protein expression.
﻿While it is believed that the great preponderance of these SNPs are harmless, there is a substantial number that have been associated with various diseases."

> "SNPs have been associated with diseases such as cystic fibrosis, Huntington's chorea, beta-thalassemia, muscular dystrophy, fibro muscular displasia, pheny ketonuria, Type II diabetes, a hyperlipidemous disorder associated with Apolipoprotein E2, at least one form of hypertension, and some forms of migraine headaches."

Basically, people who have some types of SNPs mentioned above, only have them because the alleles both had it, but if 1 didn't have the mutation, that one would be dominant.

> "While the cost of sequencing the genome of an individual human has been following a steeply declining price/performance curve, where performance is measured in terms of accuracy and time, the substantial cost that still stands today prohibits its use on a broad commercial scale"

This is one of the biggest roadblocks for an invention like this. The cost of getting these tests done and the accessibility of them.

> "In addition, not all persons possessing a given SNP in a given variant will develop a disease associated with that SNP"

Person's enviornment can impact this.

> “DNA comprising 30,000 distinct genes is transcribed into messenger RNA, which is in turn translated into the sequence of amino acids comprising potentially millions of distinct proteins, therefore adds significant complexity to associations of genotypes with phenotypes”

> “search for unknown disease-causing variants, whole-genome association scans using hundreds of thousands of genetic markers simultaneously are likely to face serious theoretical-statistical challenges”

> “While these databases contain millions of entries each including the position of the SNP on the genome, they do not provide significant phenotypic information about the SNPs at the levels which need to be reached, namely from the genome to the phenotype and the clinic.”

A future issue that we may have.

> “In addition to the scientific, technological and medical complexities that characterize the development and commercialization of genetic profile products and services, there are growing legal and regulatory complexities. For example, patient privacy has been a growing concern in multiple jurisdictions. In Europe”

Ethical and privacy issues

> “a large, dynamic, well-curated, database containing the associations between diverse genotypes and phenotypes must be maintained, easily accessed, and updated at very frequent intervals;”

The database, of course, needs to be updated constantly with accurate and real-world data from around the world.

> “In accordance with the method, the markers are compared to a multivariate scoring matrix to obtain a marker score, from which it is determined whether an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes is indicated”

Probability system

> “Third, the wild-type or original sequences and the mutated sequences are scored with one or more predictive models selected from the universe of: statistical models, such as weight matrix, Baysian, hidden markov models, semi- or general hidden Markov models; artificial intelligence models; and discriminative and predictive models related to binding affinity. This process results in splicing strength scores for the original variant and the variant with the SNP.”

**Comment by VerisimilitudeX** [comment]
@LimesKey could you please copy this over the #research in the Discord server?

**Reply by LimesKey** [reply]
Yes

**Comment by VerisimilitudeX** [comment]
@LimesKey I don't have access to Discord right now (on vacation) and we're going to have quite a few new contributors joining the server and repository. Could you please onboard them and add some more issues/features for them to work on until I come back (e.g., introducing each new member to DNAnalyzer and adding a new feature on the thing that @Karzinisierung was talking about in #research)? Thanks again!

**Reply by VerisimilitudeX** [reply]
@LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey @LimesKey

**Reply by LimesKey** [reply]
Ah

**Reply by LimesKey** [reply]
Sure thing, will do

**Reply by VerisimilitudeX** [reply]
Alright, thank you so much!

**Reply by VerisimilitudeX** [reply]
Happy holidays!! 🎄🎉
